.vale/styles/RedHat/SimpleWords.yml

```diff
@@ -12,7 +12,6 @@ swap:
   "objective(?! C?)": aim|goal
   absent: none|not here
   abundance: plenty
-  accelerate: speed up
   accentuate: stress
   accompany: go with
   accomplish: carry out|do
```

Comment:
This is another contradiction. Carry out is a phrasal verb, so accomplish would actually be preferred. Do is a good alternative though 

@aireilly @gaurav-nelson @Chandralekha-RedHat

Reply:
Some other thoughts:

plenty: It's better to not put numerical descriptors at all (for example, what does "many applications" mean?) So maybe we shouldn't suggest abundance OR plenty. 

absent: While I agree absent is a complicated word, it is often not going to match grammatically with "none" or "not here". For instance, "If the button is absent" != "*If the button is none" or "*If the button is not here" (not there would make sense in this example)